src/backend/joanie/core/admin.py

```diff
@@ -51,7 +54,13 @@ class CourseAdmin(TranslatableAdmin):
 class CourseRunAdmin(TranslatableAdmin):
     """Admin class for the CourseRun model"""
 
-    list_display = ("title", "resource_link", "start", "end", "state")
+    list_display = ("title", "resource_link", "start", "end", "state", "is_gradable")
+    actions = ("mark_as_gradable",)
+
+    @admin.action(description=_("Mark course run as gradable"))
+    def mark_as_gradable(self, request, queryset):  # pylint: disable=no-self-use
+        """Mark selected course runs as gradable"""
+        queryset.update(is_gradable=True)
```

Comment:
Is the idea to mass mark a whole query of course runs at once?

Reply:
I used it for development purpose but I though it might be useful for an administrator to mark a selection of course runs as gradable in one go.